src/PVRIptvData.cpp

```diff
@@ -274,11 +274,24 @@ bool PVRIptvData::LoadEPG(time_t iStart, time_t iEnd)
     entry.iGenreType = 0;
     entry.iGenreSubType = 0;
     entry.strPlotOutline = "";
+    entry.strOriginalTitle = "";
+    entry.strCast = "";
+    entry.strDirector = "";
+    entry.strWriter = "";
+    entry.iYear = 0;
+    entry.strIMDBNumber = "";
+    entry.strEpisodeName = "";
     entry.startTime = iTmpStart;
     entry.endTime = iTmpEnd;
 
     GetNodeValue(pChannelNode, "title", entry.strTitle);
     GetNodeValue(pChannelNode, "desc", entry.strPlot);
+    GetNodeValue(pChannelNode, "sub-title", entry.strOriginalTitle);
+    GetNodeValue(pChannelNode, "actor", entry.strCast);
+    GetNodeValue(pChannelNode, "director", entry.strDirector);
+    GetNodeValue(pChannelNode, "writer", entry.strWriter);
+    GetNodeValue(pChannelNode, "Id", entry.strIMDBNumber);
+    GetNodeValue(pChannelNode, "episode-num", entry.strEpisodeName);
```

Comment:
Not sure what `episode-num` is defined to contain, but the name suggests this is more the number of the episode and not its name?

So, maybe mapping `episode-num` to `entry.iEpisodeNumber ` would be correct, no?

Reply:
according specs https://github.com/AlekSi/xmltv/blob/master/xmltv.dtd#L365 it can be

Reply:
That is episode number , in Kodi that tag is iEpisodeId).I didnt add that.
Mine is strEpisodename:)

Reply:
But why set episode-num from the XMLTV to the strEpisodeName in kodi pvr? 

Can someone give some examples of what episode-num fields look like from some XMLTV?

Reply:
From reading the spec this need to parsed.

Reply:
@phunkyfish i have some for czech tv stations http://phazebox.com/epg.php?user=dp&email=dp@dp.sk

Reply:
Ok, there seems to be two types of episode-num system and onscreen from a scan of the samples. The format fro onscreen does not appear to be consistent or at the very least can be language dependent.

```
<episode-num system="themoviedb.org">movie/17264</episode-num>
```

```
<episode-num system="onscreen">S7 E2</episode-num>
```